src/Analysis/Ast/Impl/Analyzer/PythonAnalyzer.cs

```diff
@@ -51,17 +51,14 @@ public sealed class PythonAnalyzer : IPythonAnalyzer, IDisposable {
         private PythonAnalyzerSession _nextSession;
         private bool _forceGCOnNextSession;
 
-        public PythonAnalyzer(IServiceManager services, string cacheFolderPath = null) {
+        public PythonAnalyzer(IServiceManager services) {
```

Comment:
If this class isn't going to add to services, you can get away with lowering this to `IServiceContainer`.